CHANGELOG/CHANGELOG-1.5.md

```diff
@@ -0,0 +1,98 @@
+- Move throughput calculation from logstash to flow-aggregator. ([#2692](https://github.com/antrea-io/antrea/pull/2692), [@heanlan])
+- Add Go version information to full version string for Antrea binaries. ([#3182](https://github.com/antrea-io/antrea/pull/3182), [@antoninbas])
+- Use go 1.17 to build release assets. ([#3007](https://github.com/antrea-io/antrea/pull/3007), [@antoninbas])
+- Improve kind-setup.sh script and Kind documentation. ([#2937](https://github.com/antrea-io/antrea/pull/2937), [@antoninbas])
+- Start enabling Go benchmark tests in CI. ([#3004](https://github.com/antrea-io/antrea/pull/3004), [@wenqiq])
+- Upgrade Windows OVS version to 2.15.2 to pick up some recent patches. ([#2996](https://github.com/antrea-io/antrea/pull/2996), [@lzhecheng]) [Windows]
+- Remove HNSEndpoint only if infra container fails to create. ([#2976](https://github.com/antrea-io/antrea/pull/2976), [@lzhecheng]) [Windows]
+- Use OVS Port externalIDs instead of HNSEndpoint to cache the externalIDS when using containerd as the runtime on Windows. ([#2931](https://github.com/antrea-io/antrea/pull/2931), [@wenyingd]) [Windows]
+- Reduce the duration of disconnection when starting antrea-agent on Windows Node by using Windows management virtual network adapter as OVS internal port. ([#3067](https://github.com/antrea-io/antrea/pull/3067), [@wenyingd]) [Windows]
+
+### Fixed
+
+- Fix error handling of the "Reject" action of Antrea-native policies when determining if the packet belongs to Service traffic. ([#3010](https://github.com/antrea-io/antrea/pull/3010), [@GraysonWu])
+- Make the "Reject" action of Antrea-native policies work in AntreaIPAM mode. ([#3003](https://github.com/antrea-io/antrea/pull/3003), [@GraysonWu])
+- Set ClusterGroup with child groups to `groupMembersComputed` after all its child groups are created and processed. ([#3030](https://github.com/antrea-io/antrea/pull/3030), [@Dyanngg])
+- Fix status report of Antrea-native policies with multiple rules that have different AppliedTo. ([#3074](https://github.com/antrea-io/antrea/pull/3074), [@tnqn])
+- Fix typos and improve the example YAML in antrea-network-policy doc. ([#3079](https://github.com/antrea-io/antrea/pull/3079), [#3092](https://github.com/antrea-io/antrea/pull/3092), [#3108](https://github.com/antrea-io/antrea/pull/3108) [@antoninbas] [@Jexf] [@tnqn])
+- Fix duplicated attempts to delete unreferenced AddressGroups when deleting Antrea-native policies. ([#3136](https://github.com/antrea-io/antrea/pull/3136), [@Jexf])
+- Add retry to update NetworkPolicy status to avoid error logs. ([#3134](https://github.com/antrea-io/antrea/pull/3134), [@Jexf])
+- Fix NetworkPolicy resources dump for Agent's supportbundle. ([#3083](https://github.com/antrea-io/antrea/pull/3083), [@antoninbas])
+- Fix dual-stack support for Vagrant-based test cluster. ([#3158](https://github.com/antrea-io/antrea/pull/3158), [@antoninbas])
```

Comment:
same comment as above, you could omit this

Reply:
removed